Sources/Components/TextView/TextView.swift

```diff
@@ -4,8 +4,22 @@
 
 import UIKit
 
-public protocol TextViewDelegate: UITextViewDelegate {
+public protocol TextViewDelegate: class {
```

Comment:
You could change to `AnyObject`, I think that is preferable even though they at the moment gives the same result :)